src/components/FormLayout/FormLayout.scss

```diff
@@ -29,6 +24,14 @@ $item-min-size: rem(220px);
   flex: 1 1 $item-min-size;
   margin-top: spacing();
   margin-left: spacing(loose);
-  min-width: $item-min-size;
   max-width: calc(100% - #{spacing(loose)});
+
+  .grouped & {
+    min-width: $item-min-size;
+  }
+
+  .condensed & {
+    flex-basis: (0.5 * $item-min-size);
```

Comment:
Why `.5`? Should we save this to a variable?

Reply:
This was like this so I opted not to touch it. It's only used once so I don't see an issue with it. Do you?

Reply:
I think this is fine as is. I think this pretty clearly communicates that it is half the item min size.

Reply:
We do the same thing with `* -1` to make a number negative. We don't add comments for that.